client/login/magic-login/index.jsx

```diff
@@ -93,7 +93,7 @@ class MagicLogin extends React.Component {
 				'magic-login': true,
 				'magic-login__request_link': ! magicLoginView,
 			} }>
-				<PageViewTracker path="/login" title="Login" />
+				<PageViewTracker path="/log-in/link" title="Login" />
```

Comment:
Thanks, @stephanethomas. I assumed that'd be the case. Have this partially implemented in #14949 ([#](https://github.com/Automattic/wp-calypso/pull/14949/files#diff-a9cff22e01ecdbb84156b39131acf6e4R74)).

To clarify -- should `title` be `Login` for all Magic Login views as well? ...or should it be distinct?